src/WinRPM.jl

```diff
@@ -510,10 +510,6 @@ function prompt_ok(question)
     end
 end
 
-function help()
```

Comment:
why delete this? someone thought it was worth adding

Reply:
Would be better to have a module docstring or a proper documentation website instead of a help function that just dumps the readme. This was added 4 years ago. I suspect Julia didn't have as good documentation features back then, which warranted this function.

Reply:
sure, but that says to me that this should be deleted after one of those is added, not before